Code/Framework/AzFramework/Tests/InputTests.cpp

```diff
@@ -48,6 +48,22 @@ namespace InputUnitTests
         AZStd::unique_ptr<InputSystemComponent> m_inputSystemComponent;
     };
 
+    ////////////////////////////////////////////////////////////////////////////////////////////////
+    TEST_F(InputTest, InputChannelId_ConstExpression_CopyConstructorSuccessfull)
+    {
+        constexpr InputChannelId testInputChannelId1("TestInputChannelId");
+        constexpr InputChannelId testInputChannelId2(testInputChannelId1);
+        EXPECT_EQ(testInputChannelId1, testInputChannelId2);
```

Comment:
To truly test for `constexpr-ness`
Add a `static_assert` here.
i.e
`static_assert(testInputChannelId1 == testInputChannelId2);`

Reply:
Done (had to make the equality comparison function constexpr also).